aerich/migrate.py

```diff
@@ -281,6 +281,39 @@ def _handle_m2m_fields(
                 if add:
                     cls._add_operator(cls.drop_m2m(table), upgrade, True)
 
+    @classmethod
+    def _handle_fk_fields(
+        cls,
+        old_model_describe: Dict,
+        new_model_describe: Dict,
+        model,
+        old_models,
+        new_models,
+        upgrade=True,
+        is_o2o=False,
```

Comment:
I think instead of the `is_o2o` flag it might be more readable to have two methods: `_handle_fk_fields` and `_handle_o2o_fields` that would be calling a third method containing the shared code. Maybe, `_handle_relational_fields` because both o2o and foreign fields are inherited from `RelationalField`?

Reply:
is_o2o is not used anymore.